terraform/helm/aptos-node/values.yaml

```diff
@@ -66,6 +66,9 @@ fullnode:
   affinity: {}
   config:
     max_inbound_connections: 100
+    shared_mempool_max_concurrent_inbound_syncs: 16
```

Comment:
@bchocho - Are these configs that should be default for FNs instead of just being for `Forge`?

Reply:
Yes, these yaml's will affect the defaults outside of Forge as well.

Reply:
Seems like txns per account bumping up 5 -> 10 didn't help? Let's dial that down in that case?